docs/enhancements/support-node-labeling.md

```diff
@@ -0,0 +1,128 @@
+---
+title: node labeling support
+authors:
+- "@oamizur"
+  creation-date: 2022-12-20
+  last-updated: 2022-12-20
+---
+
+# Support node labeling during installation
+
+## Summary
+
+There was a requirement to support node roles during installation.  These roles 
+are extra roles, which are used by the user and not by any Openshift components.
+These labels should be applied on agent CR which represents a node in a spoke cluster.
+
+## Motivation
+
+Currently, nodes can be assigned with a single role which can be one of two values [ master, worker].
+The may want to add additional roles.  These roles are for the user usage only.
+
+### Goals
```

Comment:
what about day2?

Reply:
iiuc the RFE says they are doing it today at day-2 which is something they prefer to avoid.
Let's sit with Jose and Ian to understand the details.

Reply:
It will be done through the Kube API, so the controller has access to the spoke nodes.

Reply:
I assume you mean service will have and not controller right?